pkg/rpcclient/client.go

```diff
@@ -74,6 +74,7 @@ type cache struct {
 	initDone                 bool
 	network                  netmode.Magic
 	stateRootInHeader        bool
+	millisecondsPerBlock     time.Duration
```

Comment:
Really needed?

Reply:
No, it's a remnant from the previous version, fixed.